Test Fails for Node.js >= 12

Hi, the tests fail with the following error for Node.js >= 12:
```
[master][~/ups/grunt-legacy-util]$ npm run test
(node:22763) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.

> grunt-legacy-util@1.1.1 test /home/nilesh/ups/grunt-legacy-util
> grunt test

Running "jshint:all" (jshint) task
>> 3 files lint free.

Running "nodeunit:util" (nodeunit) task
Testing index.js......(node:22777) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
.............F.........
>> util.spawn.multibyte - partial stdout
>> Error: 'こんにちは(node:22887) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.' == 'こんにちは'
>> at /home/nilesh/ups/grunt-legacy-util/test/index.js:300:12
>> at callDone (index.js:160:5)
>> at ChildProcess.<anonymous> (index.js:199:5)
>> at ChildProcess.emit (events.js:210:5)
>> at ChildProcess.EventEmitter.emit (domain.js:475:20)
>> at maybeClose (internal/child_process.js:1021:16)
>> at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)

Warning: 1/98 assertions failed (2468ms) Use --force to continue.

Aborted due to warnings.
npm ERR! code ELIFECYCLE
npm ERR! errno 6
npm ERR! grunt-legacy-util@1.1.1 test: `grunt test`
npm ERR! Exit status 6
npm ERR! 
npm ERR! Failed at the grunt-legacy-util@1.1.1 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/nilesh/.npm/_logs/2020-01-24T19_25_04_713Z-debug.log
```

It would be great if they could be fixed soon. Thanks!

--- a/index.js
+++ b/index.js
@@ -183,16 +183,16 @@ util.spawn = function(opts, done) {
   }
 
   var child = spawn(cmd, args, opts.opts);
-  var stdout = new Buffer('');
-  var stderr = new Buffer('');
+  var stdout = Buffer.from('');
+  var stderr = Buffer.from('');
   if (child.stdout) {
     child.stdout.on('data', function(buf) {
-      stdout = Buffer.concat([stdout, new Buffer(buf)]);
+      stdout = Buffer.concat([stdout, Buffer.from(buf)]);
     });
   }
   if (child.stderr) {
     child.stderr.on('data', function(buf) {
-      stderr = Buffer.concat([stderr, new Buffer(buf)]);
+      stderr = Buffer.concat([stderr, Buffer.from(buf)]);
     });
   }
   child.on('close', function(code) {

--- a/test/fixtures/spawn-multibyte.js
+++ b/test/fixtures/spawn-multibyte.js
@@ -4,7 +4,7 @@
 // A multibyte buffer containing all our output. We will slice it later.
 // In this case we are using a Japanese word for hello / good day, where each
 // character takes three bytes.
-var fullOutput = new Buffer('こんにちは');
+var fullOutput = Buffer.from('こんにちは');
 
 // Output one full character and one third of a character
 process.stdout.write(fullOutput.slice(0, 4));

--- a/test/index.js
+++ b/test/index.js
@@ -477,10 +477,10 @@ exports['util.recurse'] = {
   'buffer': function(test) {
     test.expect(1);
     var actual = util.recurse({
-      buf: new Buffer('buf'),
+      buf: Buffer.from('buf'),
     }, this.typeValue);
     var expected = {
-      buf: {type: 'buffer', value: new Buffer('buf')},
+      buf: {type: 'buffer', value: Buffer.from('buf')},
     };
     test.deepEqual(actual, expected, 'Should not mangle Buffer instances.');
     test.done();

